spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/DnnGraph.scala

```diff
@@ -394,14 +394,15 @@ class DnnGraph(
    */
   private def fusion(): Unit = {
     if (!this.train) {
-      for (j <- 0 to 3) {
+      for (j <- 0 to 4) {
         var i = forwardExecution.length - 1
         while (i >= 0) {
-          if (j == 0) Fusion.fuseModule(forwardExecution(i))
+          if (j == 0) Fusion.fuseScale(forwardExecution(i))
```

Comment:
So, `scale` is not fused in our previous release?

Reply:
0.10 only fuse the bn + scale. But has no bn + scale + relu fusion.